packages/pharos-cli/src/templates/typescript-file-template.js

```diff
@@ -5,6 +5,8 @@ import type { TemplateResult, CSSResultArray } from 'lit';
 import { ${camelCaseName}Styles } from './pharos-${componentName}.css';
 
 export class Pharos${titleCaseName} extends PharosElement {
+  componentName = "Pharos${titleCaseName}";
```

Comment:
```suggestion
  static componentName = "Pharos${titleCaseName}";
```
Can we not set these to `static`?

Reply:
I tried, but TBH I was having some trouble getting it all to work. I could very well be missing something.

I was trying to put `static` [here in the base class](https://github.com/ithaka/pharos/blob/80bab8684d65b15dc32816fcd957e2dab9dff5dd/packages/pharos/src/utils/mixins/pharos-component.ts#L14), but it kept coming back `undefined` when I would try to do `this.dataset.pharosComponent = this.componentName;`

Then each of the components I also tried putting `static`, but it didn't seem to have any effect. It wasn't very consistent either because some components that inherit from one another like `PharosSidenavLink` required an `override` and others didn't which surprised me. I kind of figured it might have something to do with the fancy things we are doing in some components on the other side of the `extends` (ie. `ScopedRegistryMixin(...)`.